csr2transmart/mappers/ontology_mapper.py

```diff
@@ -49,8 +49,9 @@ def map_concept_node(self, node: OntologyConfigTreeNode) -> ConceptNode:
         concept = Concept(node.concept_code, node.name, concept_path, concept_type)
         self.concept_code_to_concept[node.concept_code] = concept
 
-        metadata_value: Dict[str, str] = {
-            'subject_dimension': self.entity_name_to_subject_dimension_value(entity_name)}
+        metadata_value: Dict[str, str] = dict()
```

Comment:
Suggestion for code structure improvement: replace these lines with a single call
```python3
concept_node.metadata = get_metadata_for_entity(entity_name)
```
and combine the filtering for subject entities with the exception for the patient dimension (in `entity_name_to_subject_dimension_value`) in one function.